storybook/webpack.config.js

```diff
@@ -1,30 +1,33 @@
-const findCacheDir = require('find-cache-dir');
-
 module.exports = {
```

Comment:
You can use ES2015 module syntax here. Why not webpack 3?

Reply:
No in the config it's normal node.js code. Latest storybook is using webpack 2 😉